### PR TITLE
RN-1094: export dashboard settings

### PR DESCRIPTION
**Pull request by alexd-bes**
### Changes:
- Added export with labels and export with table options to whole dashboard export

---

### Screenshots:
Screen Shot 2024-01-18 at 2 56 19 PM

### CODE REVIEW

File: packages/tupaia-web/src/features/Dashboard/ExportDashboard/ExportConfig.tsx [hunk 1]
```diff
@@ -56,12 +55,12 @@ const ExportSettingsContainer = styled.div`
...
-  margin-bottom: 2rem;
+  margin-block-end: 2rem;
 
   @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
     width: 62%;
-    margin-right: 2rem;
-    margin-bottom: 0;
+    margin-inline-end: 2rem;
+    margin-block-end: 0;
   }
```
Comment (jaskfla) [inline packages/tupaia-web/src/features/Dashboard/ExportDashboard/ExportConfig.tsx:64]: This is more of a question than anything: Do we have an established pattern for whether to:

- style for desktop, and override when on mobile; versus
- style for mobile, and override for bigger windows.

I’ve seen in a few places, like here, where desktop is the “default”; but it’s arguably more failsafe to style first for mobile, and selectively style for desktop. In theory if we’re 100% rigorous the result is identical, but this way feels a little more fail-safe

File: packages/tupaia-web/src/features/ExportSettings/DisplayOptionsSettings.tsx [hunk 2]
```diff
@@ -24,8 +24,8 @@ const Checkbox = styled(BaseCheckbox)`
 
 const Group = styled(FormGroup)`
   > div {
```
Comment (jaskfla) [inline packages/tupaia-web/src/features/ExportSettings/DisplayOptionsSettings.tsx:26]: What are the `<div>`s this is selecting? If it’s just the checkboxes, then `> ${Checkbox}` might be preferable to `> div`

Alternatively (and maybe better?): Consolidate all the styling of `Checkbox` in its definition:

```css
/* const Checkbox = styled(BaseCheckbox) ... */
${FormGroup} > & {
  margin-inline-start: 0.5rem;
  margin-block-start: 0;
}
```

File: packages/tupaia-web/src/features/ExportSettings/DisplayOptionsSettings.tsx [hunk 3]
```diff
@@ -24,8 +24,8 @@ const Checkbox = styled(BaseCheckbox)`
...
-    margin-left: 0.5rem;
-    margin-top: 0;
+    margin-inline-start: 0.5rem;
+    margin-block-start: 0;
   }
 `;
```
Comment (jaskfla) [inline packages/tupaia-web/src/features/ExportSettings/DisplayOptionsSettings.tsx:30]: What are the `<div>`s this is selecting? If it’s just the checkboxes, then `> ${Checkbox}` might be preferable to `> div`

Alternatively (and maybe better?): Consolidate all the styling of `Checkbox` in its definition:

```css
/* const Checkbox = styled(BaseCheckbox) ... */
${FormGroup} > & {
  margin-inline-start: 0.5rem;
  margin-block-start: 0;
}
```

**Review by jaskfla — APPROVED** [review]
One truly minuscule nitpick (which predated this PR), but makes sense to me!

**Comment by jaskfla** [comment]
Any opinion on “include labels/table” instead of “export with labels/table”?

**Comment by alexd-bes** [comment]
> Any opinion on “include labels/table” instead of “export with labels/table”?

This one is because our `presentationOptions` in the codebase have these set as `exportWithXXX` for the default values

**Pull request merged**